azure-pipelines-conditional-integration.yml

```diff
@@ -3,152 +3,148 @@
 #
 
 variables:
```

Comment:
These files auto-formatted for me when saving in VS Code. No actual changes except for getting rid of yarn install